packages/fxa-content-server/server/lib/routes/react-app/index.js

```diff
@@ -66,6 +66,7 @@ const getReactRouteGroups = (showReactApp, reactRoute) => {
         'confirm',
         'confirm_signup_code',
         'primary_email_verified',
+        'signup',
```

Comment:
This is on line 65 too

Reply:
Oh might've been from a rebase... I started the branch long before signup stuff from reaction conversion got merged into main so I had to add it manual at that time.  Thanks for catching it.